maintenance/idr0021-data-prep.md

```diff
@@ -0,0 +1,90 @@
+
+Workshop data preparation (idr0021)
+===================================
+
+This document details the steps to prepare data from IDR for a workshop demonstrating
+analysis with Fiji, usage of Map Annotations and OMERO.tables and filtering with OMERO.parade.
+We use IDR0021, which is a Project containing 10 Datasets with a total of ~400 Images.
+
+
+Download IDR data
+=================
+
+External users
+--------------
+
+Please contact the OME team for access to download IDR data.
+
+Internal users
+--------------
+
+Follow instructions on the
+[IDR submission workflow](https://docs.google.com/document/d/1TmBZ43_yhiO3AOua8oMk4mPWKWJtpeYNc2KLP17h-1I/edit#) to download the data via ```ssh``` from ```idr0-slot3```.
+
+
+Prepare IDR-metadata and import
+===============================
+
+Clone https://github.com/IDR/idr-metadata and edit
+```idr-metadata/idr0021-lawo-pericentriolarmaterial/experimentA/idr0021-experimentA-filePaths.tsv```
+to point ALL paths at the location of the data downloaded above.
+e.g.
+```Dataset:name:CDK5RAP2-C	/full/path/to/data/idr0021/CDK5RAP2-C/Centrin_PCNT_Cep215_20110506/Centrin_PCNT_Cep215_20110506_Fri-1545_0_SIR_PRJ.dv```
+
+
+If you don't want to use in-place import, comment out this line in idr-metadata/bulk.yml:
+
+	transfer: "ln_s"
+
+
+Do the bulk import:
+
+	$ cd idr-metadata/idr0021-lawo-pericentriolarmaterial/experimentA/
+	$ omero import --bulk idr0021-experimentA-bulk.yml
```

Comment:
Better is ``path/to/omero import...``

Reply:
Your setup would definitely assume that you have ``omero`` on your path, because you are not in the server directory

Reply:
Now I am finding out (after attempting to do the import on ``outreach`` ) that actually I would prefer the single-repo approach of @sbesson https://github.com/sbesson/idr0021-lawo-pericentriolarmaterial/tree/master/experimentA. This is because the @sbesson 's way gets rid of the top-directory ``bulk.yml`` , which sits in the ``idr-metadata`` top directory in the approach described here. Either I woudl have to copy the whole structure of the idr-metadata to ``outreach``, which is not wise, or I would have to ``fiddle`` with ``bulk.yml`` and the study-specific yml (but this work is extra to the workflow described here, and was already achieved by @sbesson in https://github.com/sbesson/idr0021-lawo-pericentriolarmaterial/tree/master/experimentA